app/resources/api/v1/set_transaction_resource.rb

```diff
@@ -16,6 +16,21 @@ def meta(options)
         }
       end
 
+      before_create do
```

Comment:
Please could this be commented to explain why it's necessary?

I think it's for whether a set is being created/already exists

Reply:
It will be the user to set as owner for the set if a new set is created in this transaction